functions.php

```diff
@@ -1122,10 +1123,13 @@ function eypd_d3_array( $data ) {
 			unset( $unit );
 
 		}
+		if ( 0 === strcmp( 'Professional Development Certificate Credit Hours', $key ) ) {
+			$total = $total + intval( $event['hours'] );
+		}
 
 		foreach ( $cat as $k => $v ) {
 			$result[ $i ]['label'] = html_entity_decode( $k );
-			$result[ $i ]['value'] = number_format( $v['value'], 1 );
+			$result[ $i ]['value'] = number_format( $v['value'], 8 );
```

Comment:
assuming `8` is not intentional? 
![image](https://user-images.githubusercontent.com/2048170/49042226-5dfe8480-f17c-11e8-8569-f3fe8bbc5abc.png)

Reply:
hold on...it's me.